std_msgs/Empty empty
---
sentor/TopicMapArray topic_maps
bool success
